skore/tests/unit/sklearn/test_estimator.py

```diff
@@ -209,7 +210,8 @@ def test_estimator_report_from_fitted_pipeline(binary_classification_data_pipeli
     estimator, X, y = binary_classification_data_pipeline
     report = EstimatorReport(estimator, X_test=X, y_test=y)
 
-    assert report.estimator is estimator  # we should not clone the estimator
+    check_is_fitted(report.estimator)
+    assert isinstance(report.estimator, Pipeline)
```

Comment:
It is going to be reverted normally.

Reply:
When `fit=False` we deepcopy, so the identity test doesn't work

Reply:
Yep sorry I was I confused with the `clone` one ;)